ext/standard/string.c

```diff
@@ -6114,8 +6114,8 @@ PHP_FUNCTION(strpbrk)
 	ZEND_PARSE_PARAMETERS_END_EX(RETURN_FALSE);
 
 	if (!ZSTR_LEN(char_list)) {
-		php_error_docref(NULL, E_WARNING, "The character list cannot be empty");
-		RETURN_FALSE;
+		zend_throw_error(NULL, "The character list cannot be empty");
+		return;
```

Comment:
An alternative here would be to drop the warning instead and return false -- this is consistent with searching for an empty list of characters, which will never be found.

Not sure whether that's actually a good idea though.

Reply:
I don't have a strong opinion in this, maybe let internals decide?